packages/jwt-middleware/README.md

```diff
@@ -0,0 +1,31 @@
+# JSON Web Token Middleware
+
+This middleware allows to secure execution of a webtask using JWT tokens by validating issuer, audience, and scope claims.
+
+## Usage
+
+1. Set the `wt-node-dependencies` metadata property to the stringified JSON of an object with names of modules as the keys and values set to the latest version for the corresponding module.
+
+```
+{
+    "@webtask/middleware-compiler": "1.3.0", 
+    "express-jwt": "5.3.1",
+    "jws": "3.1.4",
+    "jwks-rsa": "1.2.1"
+}
+```
+
+2. Set the `wt-compiler` metadata property on your webtask to `@webtask/middleware-compiler`.
+
+3. Set the `wt-compiler` metadata property to `@webtask/jwt-middleware`.
```

Comment:
Should this really be `wt-compiler` here? You set it to `wt-compiler` in step 2

Reply:
fixed based on Tomek's comment